node/src/command.rs

```diff
@@ -155,11 +162,7 @@ pub fn run() -> sc_cli::Result<()> {
         None => {
             let runner = cli.create_runner(&cli.run)?;
             runner.run_node_until_exit(|config| async move {
-                match config.role {
-                    Role::Light => service::new_light(config),
-                    _ => service::new_full(config),
-                }
-                .map_err(sc_cli::Error::Service)
+                service::new_full(config).map_err(sc_cli::Error::Service)
```

Comment:
There is no `Role::Light` anymore?

Reply:
The light client has been removed from `node-template`